dynamiqs/mesolve/mesolve.py

```diff
@@ -46,29 +51,31 @@ def mesolve(
 
     # === solve differential equation with diffrax
     H = totime(H)
-    Ls = format_L(jump_ops)
+    Ls = [totime(L) for L in jump_ops]
```

Comment:
I don't understand why we move `totime` here

Reply:
No strong reason, I just thought it was easier to read (this way, `format_L` just takes care of the batching dimension). I can put it back if you want !

Reply:
Ok I was just curious, we can leave it here